core/bin/external_node/src/config/mod.rs

```diff
@@ -39,6 +41,8 @@ pub struct RemoteENConfig {
     pub l2_chain_id: L2ChainId,
     pub l1_chain_id: L1ChainId,
     pub max_pubdata_per_batch: u64,
+    pub l1_batch_commit_data_generator_mode: L1BatchCommitDataGeneratorMode,
+    pub dummy_prover: bool,
```

Comment:
I don't see any place where it's used, why was it added?

Reply:
I still want to keep it in genesis... because it shows the nature of the system